sys/vm/vm_cheri_revoke.c

```diff
@@ -1047,9 +1026,8 @@ vm_cheri_revoke_publish_epochs(
 {
 	struct cheri_revoke_epochs * __capability target =
 	    &info_page->pub.epochs;
-	int res = copyoutcap(ip, target, sizeof(*target));
-	KASSERT(res == 0, ("vm_cheri_revoke_publish: bad copyout %d\n", res));
-	(void)res;
+	int res __diagused = copyoutcap(ip, target, sizeof(*target));
+	KASSERT(res == 0, ("%s: bad copyout %d\n", __func__, res));
```

Comment:
```suggestion
	int res __diagused;
	
	res = copyoutcap(ip, target, sizeof(*target));
	KASSERT(res == 0, ("%s: bad copyout %d\n", __func__, res));
```

Reply:
I think I've handled this one unless I've managed to miss a change visually diffing the KASSERT line.

Reply:
I didn't intend to touch the KASSERT line (and wish GitHub's diff would show such lines as context...)